corehq/apps/app_manager/views/download.py

```diff
@@ -293,7 +295,8 @@ def create_build_files_if_necessary_handling_conflicts(is_retry=False):
             payload = convert_XML_To_J2ME(payload, path, request.app.use_j2me_endpoint)
         response.write(payload)
         if path in ['profile.ccpr', 'media_profile.ccpr'] and request.app.last_released:
-            response['X-CommCareHQ-AppReleasedOn'] = request.app.last_released.isoformat()
+            last_released = ServerTime(request.app.last_released).user_time(pytz.UTC).done().isoformat()
```

Comment:
Is the idea here that conceptually you're sending down to the phone a "server time" (a point in time on the universal timeline), but that the phone expects it to have a timezone associated with it, so you use `user_time` to associate the UTC timezone with it?

Ideally the phone would accept the "<date>T<time>Z", since shifting to a particular timezone adds no information. We usually shift to a timezone only when we want to display a date time to a user, in their particular timezone. If making that change is out of scope and we're getting around it by just always having this date look like "<date>T<time>+00:00", this is a good way to do that.

Reply:
> Is the idea here...

Yes.

> shifting to a particular timezone adds no information

You're totally right, I forget why I was hung up on wanting to send this in UTC. Updated in 20b7602f394 to use the user's time zone, so this code should look more typical.

Reply:
Oh, uh that's not what I was suggesting, sorry to cause confusion. Timezones other than UTC can be very bad at actually measuring time, so unless there's a benefit to shifting into user time, I'd advocate keeping it in UTC. I think my previous comment was made more confusing by poor markdown formatting on my end, which I've tried to fix

Reply:
Ah, I get it now, yeah, the formatting got lost. Updated to go back to using UTC but still strip microseconds.

Deferring to @shubham1g5 on how important the `<date>T<time>+00:00` formatting is.

Reply:
Not sure if I understand the concern here correctly though we can definitely choose to have any other format, though I would prefer timezone to be part of it even if it's `+00:00` always at least in server client interactions to remove any ambiguity on client side regarding what timezone the date is in.